src/main/index.js

```diff
@@ -77,20 +91,27 @@ function onReady (err, results) {
     })
   }
 
+  if (app.rc['theme']) {
+    log.info(`theme: trying to load theme from '${app.rc['theme']}'`)
+    if (fs.existsSync(app.rc['theme'])) {
+      updateTheme()
+      log.info(`theme: set theme`)
+    } else {
+      log.error("theme: couldn't find file")
+    }
+  }
+
   if (app.rc['theme-watch']) {
-    log.info('theme-watch activated: ', app.rc['theme-watch'])
-    if (fs.existsSync(app.rc['theme-watch'])) {
-      fs.watchFile(app.rc['theme-watch'], (curr, prev) => {
+    log.info('theme-watch: activated', app.rc['theme-watch'])
+    if(app.rc['theme']) {
+      fs.watchFile(app.rc['theme'], (curr, prev) => {
         if (curr.mtime !== prev.mtime) {
           log.info('theme-watch: File changed reloading theme data')
+          updateTheme()
           // TODO make this more safe or disable when merging
```

Comment:
you can remove this line, it was there because I first tried loading the json with `require`